anomalib/core/callbacks/min_max_normalization.py

```diff
@@ -0,0 +1,83 @@
+"""Anomaly Score Normalization Callback that uses min-max normalization."""
+
+# Copyright (C) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+from typing import Any, Dict
+
+import pytorch_lightning as pl
+from pytorch_lightning import Callback
+from pytorch_lightning.utilities.types import STEP_OUTPUT
+
+from anomalib.utils.normalization.min_max_normalization import normalize
```

Comment:
Similar to the comment below:
do we need to explicitly name the module as `min_max_normalization`. It's already located in `normalisation` package. I think we could rename it as just `min_max`. in this case, it would be like `from anomalib.utils.normalization.min_max import normalize, standardize`. It would also be slightly less verbose.